container_workflow_tool/main.py

```diff
@@ -521,24 +522,19 @@ def pull_upstream(self):
         Additionally runs a script against each repository if check_script is set,
         checking its exit value.
         """
-        tmp = self._get_tmp_workdir()
-        self._change_workdir(tmp)
-        images = self._get_images()
+        tmp, images = self.preparation()
         for i in images:
             # Use unversioned name as a path for the repository
             ups_name = i["name"].split('-')[0]
-            self.distgit._clone_upstream(i["git_url"],
-                                         ups_name,
-                                         commands=i["commands"])
+            self.git_ops.clone_upstream(i["git_url"], ups_name, commands=i["commands"])
```

Comment:
Would be nice to have a better name than `i` here.

Reply:
Fixed by https://github.com/sclorg/container-workflow-tool/pull/71/commits/d1623aad6649f01099cadc67adbb67c6ed8df53a